src/languages/fr.ts

```diff
@@ -1638,23 +1638,23 @@ const translations: TranslationDeepObject<typeof en> = {
                 // eslint-disable-next-line default-case
                 switch (actorType) {
                     case CONST.NEXT_STEP.ACTOR_TYPE.CURRENT_USER:
-                        return `En attente que <strong>vos</strong> notes de frais soient automatiquement soumises${formattedETA}.`;
+                        return `En attente que <strong>vos</strong> notes de frais soient envoyées automatiquement${formattedETA}.`;
                     case CONST.NEXT_STEP.ACTOR_TYPE.OTHER_USER:
                         return `En attente que les dépenses de <strong>${actor}</strong> soient automatiquement soumises${formattedETA}.`;
                     case CONST.NEXT_STEP.ACTOR_TYPE.UNSPECIFIED_ADMIN:
-                        return `En attente de la soumission automatique des notes de frais d’un administrateur${formattedETA}.`;
+                        return `En attente que les dépenses d’un administrateur soient automatiquement soumises${formattedETA}.`;
```

Comment:
Looks like some of the updates in international files don't correspond to an update in the `en.ts` file - should we exclude those?